examples/responsive.html

```diff
@@ -0,0 +1,147 @@
+          display: none;
+        }
+      }
+      @container (min-width: 590px) {
+        .centered-controls-overlay {
+          display: none;
+        }
+        media-control-bar {
+          display: flex;
+        }
+      }
+
+      media-controller .centered-controls-overlay {
+        align-self: stretch;
+      }
+      [slot='centered-chrome'] {
+        margin: 0 15%;
+        --media-control-hover-background: none;
+        --media-control-background: none;
+      }
+      [slot='centered-chrome']:is(media-play-button, media-seek-backward-button, media-seek-forward-button) {
+        padding: 0px;
+      }
+      [slot='centered-chrome']media-play-button {
+        width: 20%;
+      }
+      [slot='centered-chrome']:is(media-seek-backward-button, media-seek-forward-button) {
+        width: 15%;
+      }
+
+      media-loading-indicator {
+        position: absolute;
+        inset: 0;
+      }
+
+      media-airplay-button[media-airplay-unavailable],
+      media-fullscreen-button[media-fullscreen-unavailable],
+      media-volume-range[media-volume-unavailable],
+      media-pip-button[media-pip-unavailable] {
+        display: none;
+      }
+
+      .examples {
+        margin-top: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <main>
+      <h1>Responsive Media Chrome</h1>
+      <media-controller>
+        <video
+          slot="media"
+          src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
+          muted
+          crossorigin="anonymous"
+          playsinline
+        >
+          <track
+            label="thumbnails"
+            default
+            kind="metadata"
+            src="https://image.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/storyboard.vtt"
+          />
+          <track
+            label="English"
+            kind="captions"
+            srclang="en"
+            src="./vtt/en-cc.vtt"
+          />
+        </video>
+        <media-poster-image
+          slot="poster"
+          src="https://image.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/thumbnail.jpg"
+          placeholder-src="data:image/jpeg;base64,/9j/2wBDAAUDBAQEAwUEBAQFBQUGBwwIBwcHBw8LCwkMEQ8SEhEPERETFhwXExQaFRERGCEYGh0dHx8fExciJCIeJBweHx7/2wBDAQUFBQcGBw4ICA4eFBEUHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh7/wAARCAASACADASIAAhEBAxEB/8QAGgABAAIDAQAAAAAAAAAAAAAAAAMEAgUGCP/EACkQAAEDAgMIAgMAAAAAAAAAAAEAAgMEBgUREgcUITFSkZTRQaEiscH/xAAYAQACAwAAAAAAAAAAAAAAAAAABQIDBv/EAB0RAAICAQUAAAAAAAAAAAAAAAABAgMFERUxwfD/2gAMAwEAAhEDEQA/AOZh2P2k/LOhq/Lf7VuPYvZxLQ6iqgXchvrxn9rpY7ojYCBU0IJ5HU3h9rU3NcGJVcVNJh2K4fDPTztlbm5reGRDhnxIzBPwkUc9RJ6dDHaLYojj2HWYeeH1nmSe1OzYXZJ54fW+ZJ7VeWrbO4SPuedpI/IOnB/TgsxJh4yIuGYu+TvAH9UXnafItWJmuTy1oZ0t7JoZ0t7Ii0InGhnS3smhnS3siIA//9k="
+        ></media-poster-image>
+        <media-loading-indicator media-loading slot="centered-chrome" no-auto-hide></media-loading-indicator>
+        <div slot="centered-chrome" class="centered-controls-overlay">
```

Comment:
nitpick(non-blocking): Since we have to declare `slot="centered-chrome"`, is the class redundant? Should we just use an attribute selector here as our "recommended approach"?

Reply:
It's still helpful because otherwise the selector is `[slot=centered-chrome]:not(media-loading-indicator)` or something. FYI, copied from the mobile controls example.